src/types/user.ts

```diff
@@ -516,6 +516,23 @@ export default /* GraphQL */ `
     id: ID!
     address: String!
     createdAt: DateTime!
+
+    """NFT assets owned by this wallet address"""
+    assets: [NFTAsset!]!
+  }
+
+  """ NFT Asset """
+  type NFTAsset {
+    tokenId: String!
+    name: String!
+    description: String
+    imageUrl: String!
```

Comment:
what are the differences among `image*`? I think we can just keep a thumb/preview (for client-side preview) and an original (for upload back to us via `singleUploadFile` mutation)?

Reply:
sample from https://rinkeby-api.opensea.io/api/v1/assets?asset_contract_address=0xae89d81ab5c668661200fa9c6ed45fe1707f7097&owner=0x08f3BCa6DeF9CdF5EeE1114Ed4d2A0d91cFE4FFA&order_direction=desc&offset=0&limit=20

image_original_url is ipfs original, image_url is cached original size, preview is 250w, thumbnail is 128w

```json
    "image_url": "https://lh3.googleusercontent.com/2D6pKJyKZQ5-FfvwsgofxJp2zEuhEji7iI06Odz-NLqYK1zaQvEbIe0R1M-refwv6la8acbA6ap7k5PxtkbgFNkZHe_eUsl3Wvh9mw",
    "image_preview_url": "https://lh3.googleusercontent.com/2D6pKJyKZQ5-FfvwsgofxJp2zEuhEji7iI06Odz-NLqYK1zaQvEbIe0R1M-refwv6la8acbA6ap7k5PxtkbgFNkZHe_eUsl3Wvh9mw=s250",
    "image_thumbnail_url": "https://lh3.googleusercontent.com/2D6pKJyKZQ5-FfvwsgofxJp2zEuhEji7iI06Odz-NLqYK1zaQvEbIe0R1M-refwv6la8acbA6ap7k5PxtkbgFNkZHe_eUsl3Wvh9mw=s128",
    "image_original_url": "https://ipfs.io/ipfs/QmQvL122xLjLDWwjAJjRNkKWQsh9GzQnkWU5mtc2FPgKnY",
```